docs/admin-guide.md

```diff
@@ -666,6 +669,9 @@ behind `work.example.com`:
 > ssh root@database.work.example.com
 ```
 
+!!! tip "WARNING":
+    You can not connect to Teleport node by IP addresses using most version of OpenSSH, you can only connect via DNS name. When validating a certificate the OpenSSH client validates the DNS hostname you wish to connect to (in this case an IP address) with the `Principal` on the certificate. Newer versions of OpenSSH do allow passing in `HostKeyAlias` when connecting via IP addresses to by-pass this check, if you can only connect by IP address check if your version of OpenSSH has support for `HostKeyAlias`.
```

Comment:
* formatting is a bit off compared to the rest of the text
* most version -> most recent?